dotnet/targets/Xamarin.Shared.Sdk.targets

```diff
@@ -1180,17 +1182,37 @@
 	</PropertyGroup>
 
 	<Target Name="_ComputeDynamicLibrariesToReidentify">
-		<PropertyGroup>
-			<_ExecutablePathPrefix Condition="'$(_PlatformName)' == 'iOS' Or '$(_PlatformName)' == 'tvOS'">@executable_path/</_ExecutablePathPrefix>
-			<_ExecutablePathPrefix Condition="'$(_PlatformName)' == 'macOS' Or '$(_PlatformName)' == 'MacCatalyst'">@executable_path/../$(_CustomBundleName)/</_ExecutablePathPrefix>
+		<!-- iOS and tvOS -->
+		<PropertyGroup Condition="'$(_PlatformName)' == 'iOS' Or '$(_PlatformName)' == 'tvOS'">
+			<_ExecutablePathPrefix>@executable_path/</_ExecutablePathPrefix>
```

Comment:
Looks like the change for iOS and tvOS doesn't really change anything? If so, it's preferrable to not change the code.

Reply:
I think the error in the CI is because of the last commit I added to respect the subdirectories in the MonoBundle dir. 

> Looks like that didn't quite work:
> 
> ```
> Target Name=_ComputeDynamicLibrariesToPublish Project=interdependent-binding-projects.csproj
>     Errors
>         /Users/builder/azdo/_work/3/s/xamarin-macios/builds/downloads/dotnet-sdk-7.0.101-servicing.22567.23/packs/Microsoft.macOS.Sdk/13.0.1197-ci.pr.gh16706/targets/Xamarin.Shared.Sdk.targets(710,3): error MSB4096: The item "/Users/builder/azdo/_work/3/s/xamarin-macios/tests/xharness/tmp-test-dir/interdependent-binding-projects3468/bin/Debug/net7.0-macos/bindings-test.resources/libtest.xcframework/macos-arm64_x86_64/libtest.a" in item list "_FileNativeReference" does not define a value for metadata "RelativePath".  In order to use this metadata, either qualify it by specifying %(_FileNativeReference.RelativePath), or ensure that all items in this list define a value for this metadata. [/Users/builder/azdo/_work/3/s/xamarin-macios/tests/xharness/tmp-test-dir/interdependent-binding-projects108/interdependent-binding-projects.csproj]
> ```

I think the error in the CI is because of the last commit I added to respect the subdirectories in the MonoBundle dir where I was using `$(RelativePath)` assuming again that the condition is checked first :) It should be fixed now